aclogview/CM_Communication.cs

```diff
@@ -83,6 +89,29 @@ public class CM_Communication : MessageProcessor {
         return handled;
     }
 
+    public class TalkDirect : Message
+    {
+        public PStringChar MessageText;
+        public uint TargetID;
+
+        public static TalkDirect read(BinaryReader binaryReader)
+        {
+            var newObj = new TalkDirect();
+            newObj.MessageText = PStringChar.read(binaryReader);
+            newObj.TargetID = binaryReader.ReadUInt32();
+            return newObj;
+        }
+
+        public override void contributeToTreeView(TreeView treeView)
+        {
+            TreeNode rootNode = new TreeNode(this.GetType().Name);
+            rootNode.Expand();
+            rootNode.Nodes.Add("MessageText = " + MessageText.m_buffer);
+            rootNode.Nodes.Add("TargetID = " + TargetID.ToString());
```

Comment:
Can you do Utility.FormatGuid(TargetID) instead of .ToString()? Otherwise looks great.